components/match2/wikis/ageofempires/match_group_input_custom.lua

```diff
@@ -131,6 +131,7 @@ function CustomMatchGroupInput._getExtraData(match)
 		headtohead = match.headtohead,
 		civdraft = match.civdraft,
 		mapdraft = match.mapdraft,
+		casters = MatchGroupInput.readCasters(match, {noSort = true}),
```

Comment:
Any perticular reason why going for noSort?

Reply:
On AoE a lot of tournaments aren't hosted by organizations but rather single persons, so to me it feels natural to have the ability to list these first when they cast in their own tournament